marlowe/src/Language/Marlowe/Core/V1/Semantics/Types.hs

```diff
@@ -315,6 +334,8 @@ instance ToJSON Environment where
   toJSON Environment{..} = object
     [ "timeInterval" .= posixIntervalToJSON timeInterval]
 
+
+
```

Comment:
Could you revert this file, please? This file has been audited, we shouldn't be making changes to it, and I think none of these functions are used. You can put them in `/marlowe/src/Language/Marlowe/Utils.hs`, also I don't think the whole `Next` folder should be under the `Semantics` folder, for the same reason. Also making changes to the semantics requires updating the formalisation and spec in `marlowe` repo to match. I see why we may want them under `V1`, but under `Semantics`, I am not sure